indykite/resource_authorization_policy.go

```diff
@@ -15,20 +15,18 @@
 package indykite
 
 import (
-	"fmt"
-
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/structure"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	"github.com/indykite/jarvis-sdk-go/config"
 	configpb "github.com/indykite/jarvis-sdk-go/gen/indykite/config/v1beta1"
-	knowledge_graphv1beta1 "github.com/indykite/jarvis-sdk-go/gen/indykite/knowledge_graph/v1beta1"
-	"google.golang.org/protobuf/encoding/protojson"
 )
 
 const (
-	authorizationPolicyJSONConfigKey = "json_config"
+	authorizationPolicyJSONConfigKey = "json"
```

Comment:
can you please rename those constants and align naming?

Probably something like `authzJSONConfigKey` and `authzTagsKey` and `authzStatusKey`.
It is shared in whole package, so it can colide with different provider sometimes